src/main/java/com/brandon3055/draconicevolution/api/modules/entities/AutoFireEntity.java

```diff
@@ -0,0 +1,25 @@
+package com.brandon3055.draconicevolution.api.modules.entities;
+
+import com.brandon3055.draconicevolution.api.config.BooleanProperty;
+import com.brandon3055.draconicevolution.api.modules.Module;
+import com.brandon3055.draconicevolution.api.modules.data.AutoFireData;
+import com.brandon3055.draconicevolution.api.modules.lib.ModuleEntity;
+
+import static com.brandon3055.draconicevolution.api.config.ConfigProperty.BooleanFormatter.ENABLED_DISABLED;
+
+public class AutoFireEntity extends ModuleEntity {
+
+    private BooleanProperty autoFireEnabled = null;
+
+    public AutoFireEntity(Module<AutoFireData> module) {
+        super(module);
+        if (module.getData() != null) {
+            addProperty(autoFireEnabled = new BooleanProperty("auto_fire", true).setFormatter(ENABLED_DISABLED));
```

Comment:
This should be something along the lines of "auto_fire_mod.enable" for less ambiguous localization. 
You would then localize this with 
```java
        helper.add("auto_fire_mod.enable",                                         "Auto Fire");
        helper.add("auto_fire_mod.enable.info",                                  "Automatically fire bow at full draw.");
```